dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/services/tasks/Ds3JobTask.java

```diff
@@ -119,7 +113,7 @@ void getTransferRates(final Instant jobStartInstant, final AtomicLong totalSent,
                     totalJobSize, sourceLocation, targetLocation).toString());
         }
 
-        updateProgress(totalSent.get() / 2 , totalJobSize);
+        updateProgress(totalSent.get() / 2, totalJobSize);
```

Comment:
Do you know why we need to divide by two?

Reply:
I have no clue, but it is done that way everywhere I'll look into it for a moment.

Reply:
If this is done everywhere it's used, let's pull it into the updateProgress method.